File: LeetCode/cpp/1317_Convert_Integer_to_the_Sum_of_Two_No-Zero_Integers.cpp
class Solution {
public:
	vector<int> getNoZeroIntegers(int n) {
		vector<int> answer;

		for (int i = 1; i < n; i++) {
			if (to_string(i).find('0') != string::npos || to_string(n - i).find('0') != string::npos) {
				continue;
			}

			answer.push_back(i);
			answer.push_back(n - i);
			break;
		}

		return answer;
	}
};

File: LeetCode/java/1317_Convert_Integer_to_the_Sum_of_Two_No-Zero_Integers.java
class Solution {
    public int[] getNoZeroIntegers(int n) {
        int[] answer = new int[2];

        for (int i = 1; i < n; i++) {
            if (String.valueOf(i).contains("0") || String.valueOf(n - i).contains("0")) {
                continue;
            }

            answer[0] = i;
            answer[1] = n - i;
            break;
        }

        return answer;
    }
}